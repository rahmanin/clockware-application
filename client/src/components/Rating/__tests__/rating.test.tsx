import {RatingStars} from "../index"
import React from 'react';
import {cleanup} from '@testing-library/react';
import { act } from "react-dom/test-utils";
import {shallow} from "enzyme"
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

Enzyme.configure({ adapter: new Adapter() });
afterEach(cleanup);

act(() => {
  it('Render RatingStars', () => {
    const component = shallow(
        <RatingStars 
          readOnly={true}
        />
    );
    expect(component).toMatchSnapshot();
  });
})
