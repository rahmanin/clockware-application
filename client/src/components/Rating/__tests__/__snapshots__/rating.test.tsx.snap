// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render RatingStars 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Rating))
    name="evaluation"
    readOnly={true}
  />
  (
  New
  )
</div>
`;
